src/main/java/com/educare/unitylend/model/Wallet.java

```diff
@@ -4,36 +4,13 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.util.UUID;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class Wallet {
-
-    private String walletid;
+    private String Walletid;
```

Comment:
Better to keep same naming conventions for a single class. Either small or camelcase. Might create problems later.